buildenv/jenkins/common/build

```diff
@@ -195,26 +196,50 @@ def checkout_pullrequest() {
             if (omr_upstream) {
                 sh "git config remote.origin.url https://github.com/eclipse/omr.git"
             }
-            checkout_pullrequest(OMR_PR)
+            checkout_pullrequest(OMR_PR,'omr')
         }
     }
 }
 
-def checkout_pullrequest(ID) {
+def checkout_pullrequest(ID,repo) {
```

Comment:
Please use all UPPERCASE variable names for consistency

Reply:
fixed